designer/server/src/test/scala/pl/touk/nussknacker/ui/api/description/MigrationApiEndpointsSpec.scala

```diff
@@ -0,0 +1,25 @@
+package pl.touk.nussknacker.ui.api.description
+
+import org.scalatest.freespec.AnyFreeSpecLike
+import org.scalatest.matchers.should.Matchers
+import pl.touk.nussknacker.test.utils.domain.ReflectionBasedUtils.findSubclassesOf
+import pl.touk.nussknacker.test.utils.domain.migration.MigrationUtils.allSubclassesContainVersionField
+import pl.touk.nussknacker.ui.api.description.MigrationApiEndpoints.Dtos.MigrateScenarioRequestDto
+
+class MigrationApiEndpointsSpec extends AnyFreeSpecLike with Matchers {
+
+  "all subclasses of MigrateScenarioRequestDto" - {
+    "contain version field of type Int" in {
```

Comment:
hmm, what if we make a slight change:
```scala
sealed trait MigrateScenarioRequestDto {
  def version: Int
}
```

It looks like after doing this, the test won't be needed

Reply:
please take a look at my improvement suggestions: 
https://github.com/TouK/nussknacker/pull/5975